src/ResourceManagement/NotificationHubs/Microsoft.Azure.Management.NotificationHubs/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,31 @@
+﻿//
+// Copyright (c) Microsoft.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Azure NotificationHubs Management Library")]
+[assembly: AssemblyDescription("Provides Microsoft Azure NotificationHubs management functions for managing the Microsoft Azure NotificationHubs service.")]
+
+[assembly: AssemblyVersion("4.0.0.0")]
+[assembly: AssemblyFileVersion("4.0.0.0")]
```

Comment:
versions should be 1.0.0. in general they should be aligned with project.json version